shared-components/statviz/utils/analytics/heap/types.ts

```diff
@@ -0,0 +1,23 @@
+type HeapEvent = Record<string, string | number>;
+
+interface IHeap {
+  track: (name: string, event: HeapEvent) => void;
+}
+
+interface IDownloadByGraphEvent extends HeapEvent {
+  graphName: string;
```

Comment:
I would extend this to also track the file format , e.g. jpeg and svg atm.
In heap there is no easy way to distinguish otherwise.